descriptors/ModuleDescriptor-template.json

```diff
@@ -51,6 +51,10 @@
     {
       "id": "organizations-storage.organizations",
       "version": "1.0"
+    },
+    {
+      "id": "finance-storage.funds",
```

Comment:
please add the corresponding permission "finance-storage.funds.collection.get" to POST API module permissions.

Reply:
permission added